CHANGELOG.md

```diff
@@ -1,29 +1,14 @@
 # Lotus changelog
 
-# UNRELEASED
+# v1.28.0 / 2024-07-23
+This is the MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
 
-## ☢️ Upgrade Warnings ☢️
-
-- This Lotus release includes some correctness improvements to the events subsystem, impacting RPC APIs including `GetActorEventsRaw`, `SubscribeActorEventsRaw`, `eth_getLogs` and the `eth` filter APIs. Part of these improvements involve an events database migration that may take some time to complete on nodes with extensive event databases. See [filecoin-project/lotus#12080](https://github.com/filecoin-project/lotus/pull/12080) for details.
-
-## New features
+**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.** 
 
-- feat: Add trace transaction API supporting RPC method `trace_transaction` ([filecoin-project/lotus#12068](https://github.com/filecoin-project/lotus/pull/12068))
-
-## Improvements
+Compared to `Lotus v1.28.0-rc5`, the `Lotus v1.28.0` release:
```

Comment:
We're referencing rc5, but those notes aren't in the CHANGELOG.

Reply:
Hmmm.. Yeah, this is an artefact caused be us just updating the same entry in the Changelog for each RC, instead of copying the notes before updating rcX.